tests/test_tutorial/test_sql_databases/test_tutorial001.py

```diff
@@ -12,6 +12,11 @@
 from tests.utils import needs_py39, needs_py310
 
 
+@pytest.fixture(autouse=True)
+def deactivate_blockbuster(blockbuster):
```

Comment:
Same as before but here it's `tutorial001` `on_startup` which does a blocking call to `create_db_and_tables`